src/Microsoft.AspNetCore.Http.Abstractions/Authentication/AuthenticateInfo.cs

```diff
@@ -9,21 +9,25 @@ namespace Microsoft.AspNetCore.Http.Authentication
     /// <summary>
     /// Used to store the results of an Authenticate call.
     /// </summary>
+    [Obsolete("See https://go.microsoft.com/fwlink/?linkid=845470")]
```

Comment:
Yeah this was mostly a placeholder for now, we have some time to craft the final message, since I can't check in this PR until I've updated all the downstream repos